apps/omg_watcher/priv/repo/migrations/20190806111817_alter_txoutputs_ethevents_make_many_to_many_relation.exs

```diff
@@ -62,13 +62,9 @@ defmodule OMG.Watcher.DB.Repo.Migrations.AlterTxOutputsTableAddRootchainTxnHashD
       remove(:creating_deposit)
       remove(:spending_exit)
 
-      timestamps([type: :utc_datetime])
+      timestamps(type: :utc_datetime, default: fragment("(now() at time zone 'utc')"), null: false)
```

Comment:
Referring to [comment](https://github.com/omisego/elixir-omg/issues/966#issuecomment-531442169)
I think migration should handle the existing data by inserting `now()` to existing rows and then set `not null` constraint.
Therefore we'll have the same db on dev and prod. What do you think @jrhite :question:

Reply:
I would just add that 
`I think migration should handle the existing data by inserting now()` 
now(), might not be a good idea, because we wouldn't know which are valid dates and which are not. So maybe `1 January 1970 00:00:00 UTC` then we at least know it's not a TRUE value.

Reply:
Yeah, postgres won't behave like this for existing rows, only newly inserted rows. If we are planning to wipe staging/dev again and I would like to slip the not null constraint back in. 😉

Reply:
> Postgres doesn't know the correct the column values to insert for existing rows.
So that's why I suggested to: 
1. add a column without constraint, 
2. set some value in existing rows (if not now() then maybe epoch as @InoMurko said)
3. bring the constraint 
So we'd have the constraint and consistent DBs in each env.

> For example, for auditing purposes, or perhaps the UI wants to filter things based on created or updated date.
What's wrong with the above then? The existing rows would have the earliest date. How they will be sorted / audited with `NULL` values - why it's better

Reply:
Yes, I understand the process and steps. I can implement everything you say in this single migration.

There is no meaningful value to use, but I agree that if we're going to use any value the epoch is best.